conda-recipe/meta.yaml

```diff
@@ -14,96 +14,103 @@
 # limitations under the License.
 #===============================================================================
 
-{% set version = '2024.1' %}
+# NB: this recipe should be synced between sklearnex repo and feedstocks
+
+{% set name = "scikit-learn-intelex" %}
+{% set version = "2025.1" %}
```

Comment:
I suggest to use git env variables here
https://docs.conda.io/projects/conda-build/en/latest/user-guide/environment-variables.html#git-environment-variables

Reply:
Added